src/subscript/co2containment/calculate.py

```diff
@@ -0,0 +1,218 @@
+"""CO2 calculation methods"""

```

Comment:
Please rename the directory to `subscript/co2_containment` match the entry script filename and add an empty `__init__.py` file as the other scripts have.

subscript has the (intentional or not, perhaps dubious) convention of matching package/module names `subscript.script_a.script_a`